Readme.md

```diff
@@ -7,28 +7,46 @@
 # Tracee: Runtime Security and Forensics using eBPF
 
 Tracee uses eBPF technology to tap into your system and give you access to hundreds of events that help you understand how your system behaves.
-In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. You can also easily add your own events using the popular [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) language.
+In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. 
 Tracee provides a rich filtering mechanism that allows you to eliminate noise and focus on specific workloads that matter most to you.
 
-To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee).
+**Key Features:**
+* Kubernetes native installation
+* Hundreds of default events
+* Up-to-date tracking of malicious behaviour through signatures provided by the Aqua Security Research Team 
+* Easy configuration through Tracee Policies 
```

Comment:
in addition to easy, policies also allows more advanced and accurate tracing than before. maybe "Accurate and flexible tracing through Tracee Policies" or similar, wdyt?